docs/src/guidelines.md

```diff
@@ -7,6 +7,8 @@ Depth = 3
 
 ## Source-code guidelines
 
+- Use 4 space characters for indentation. If you use the tab character, it should appear as 4-spaces-wide.
```

Comment:
```suggestion
- Use 4 space characters for indentation.
```
No tabs, please.

Reply:
To be clear: "If you use the tab key... " ? For example, i use the TAB key in Atom which corresponds to 4 spaces under `Editor -> Tab` settings and choosing `soft` Tab Type.

Reply:
I would not explain how to set up an editor, especially it was confusing to me. All that matters is that the result has four spaces. How that happened - I don't care :wink: